index.html

```diff
@@ -3045,7 +3069,16 @@ <h3>Trust Model</h3>
 [=Issuer=] does not need to know or trust the [=verifier=].
           </li>
         </ul>
-
+       <p>
+How [=verifiers] decide which [=issuers] to trust is out of the scope of 
+this recommendation. However some [=issuers], e.g. well known organisations, 
+may be implicitly trusted by many [=verifiers] because of their reputation. 
+Some [=issuers] and [=verifiers] may be members of a specific community in which
+all community members trust each other as a result of the rules of membership. 
+Some [=verifiers] may trust a specific trust service provider whose 
+responsibility is to vet [=issuers] and list them in a trust list e.g. 
+as specified in ETSI TS 119 612 [[ETSI]].
```

Comment:
```suggestion
How [=verifiers=] decide which [=issuers=] to trust, and for what data or
purposes, is out of scope for this recommendation. Some [=issuers=] (such
as well-known organizations), might be trusted by many [=verifiers=] simply
because of their reputation. Some [=issuers=] and [=verifiers=] might be
members of a community in which all members trust each other due to the
rules of membership. Some [=verifiers=] might trust a specific trust-service
provider whose responsibility is to vet [=issuers=] and list them in a trust
list such as those specified in [[[ETSI]]] [[ETSI]].
```